src/conv-core/convcore.C

```diff
@@ -979,6 +984,10 @@ void CmiTimerInit(char **argv)
   CpvAccess(inittime_wallclock) = times(&temp);
   CpvAccess(inittime_virtual) = temp.tms_utime + temp.tms_stime;
   CpvAccess(clocktick) = 1.0 / (sysconf(_SC_CLK_TCK));
+
+#if CMK_ERROR_CHECKING
```

Comment:
Is there some other place in convcore.C (or maybe in conv-conds.C) that we could initialize this once, rather than the 8 separate times we have here?

Reply:
Yeah, I think I can move it to `CcdModuleInit` since it's used for Ccd functionality (instead of timers).